### ISSUE TITLE
Support target platform AnyCPU

### ISSUE THREAD

**Issue by hongkilldong** [bricelam/mod_spatialite-NuGet#5]
Hi
 
May it would be good to upgrade package and adding msbuild script to support of automatic deployment of package binaries not only for netcore apps. It doesn't happen when app is targeting for some of full NET framework.
Here is the mod_spatialite.targets script

[mod_spatialite.targets.txt](https://github.com/bricelam/mod_spatialite-NuGet/files/2496706/mod_spatialite.targets.txt)

It should be put into package's /build folder as mod_spatialite.targets.

**Comment by bricelam** [comment]
If you're using NuGet via `PackageReference` (not `packages.config`) ***and*** your target platform isn't AnyCPU, the package will work.

Are you using `packages.config`? or targeting AnyCPU? or both?

**Comment by hongkilldong** [comment]
Yes, i am using both as package.config and targeting project to AnyCPU. But it seems defaults for any of WPF aware projects.

**Comment by bricelam** [comment]
Clever. I just (re)noticed that you use the build machine's platform to decide on AnyCPU.

A more complete solution is to put both sets of binaries on disk and provide an API to load the appropriate one at run time. (We did this in the [Microsoft.SqlServer.Types](https://www.nuget.org/packages/Microsoft.SqlServer.Types) package)

I like your solution better because it's easier and "just works" most of the time. (Until you copy it to another machine with a different architecture)
